### PR TITLE
Contact in the footer 

### PR DESCRIPTION
**Pull request by boo-shehab**
In this request, we made the footer contact and we contact it with the Formspree:

- My partner has added the HTML structure of the form and validation

- I have added the styling of the form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -175,5 +175,21 @@ <h3 class="view">Skills <img src="img/Normal Button/Tertiary/Icons/Union.png" al
...
+        <div class="contain">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xyyvwqrd" method="POST">
+                <input type="text" required name="full-name" placeholder="full name">
+                <input type="email" required name="Email" placeholder="email">
+                <textarea name="Message" placeholder="write your message here."></textarea>
+                <button type="submit" class="normal-button">Get in touch</button>
+            </form>
```
Comment (christianbiring1) [inline index.html]: Kindly make sure your input fields are validated. 
- [ ] name field must accept up to 30 chars. `hint: max-length="30"`
- [ ] Message field accepts up to 500 chars.
- [ ]  All the fields must have the `required` attribute

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @boo-shehab  and @Moise-code , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856

## Highlights
- [ ] All linter tests are green
- [ ] Descriptive PR `title`

## Required Changes ♻️
- [ ] Kindly make sure you document your project in a professional way., All people whom work on the project must figure in the README 
- [ ] Kindly make sure the description of the repository is clear 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Writing a descriptive PR 'summary'

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by boo-shehab** [comment]
@christianbiring1 Is it good now?

File: index.html [hunk 2]
```diff
@@ -175,5 +175,21 @@ <h3 class="view">Skills <img src="img/Normal Button/Tertiary/Icons/Union.png" al
...
+    <!-- Start contact -->
+    <footer>
+        <div class="contain">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xyyvwqrd" method="POST">
+                <input type="text" required name="full-name" placeholder="full name">
+                <input type="email" required name="Email" placeholder="email">
+                <textarea name="Message" placeholder="write your message here."></textarea>
```
Comment (ghost) [inline index.html]: - [ ] As the previous reviewer requested, Kindly  an attribute `maxlength="30"` to your name input for validation
- [ ] Kindly add an attribute `maxlength="500"` to your message input for validation reasons

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @boo-shehab , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


YesYouAreGIF (2)


## Highlights

✅ Correct use of Github workflow
✅ PR Description is okey
✅ README is okay


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (boo-shehab) [reply index.html]: I did them but i think I might forget to push them I will do it now

**Comment by boo-shehab** [comment]
@frank1738 can you see it now, please

**Comment by ghost** [comment]
> @frank1738 can you see it now, please

Great work @boo-shehab . Kindly submit your project on the dashboard for another review.

**Review by ichala — APPROVED** [review]
Hi @boo-shehab ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes implemted ✔️ 

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


*************************************
# Optional Suggestions 

**_🛑 🛑 🛑 OPTIONAL BUT HIGHLY RECOMMENDED 🛑 🛑 🛑_**
Great Job doing all the requested changes , please note that you have conflicts that need to be resolved , its a reason to request changes but since you don't have any extra reviews I see that you deserve to be **_approved_** for your hard work ,but kindly **fix the conflicts** , keep rocking 👏 




_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**